src/invoices/Voucher/utils.js

```diff
@@ -0,0 +1,13 @@
+export const groupByExternalAccNumber = voucherLines => (
+  voucherLines.reduce(
+    (acc, line) => ({
+      ...acc,
+      [line.externalAccountNumber]: (acc[line.externalAccountNumber] || []).concat(line),
+    }),
+    {},
+  )
+);
+
+export const getTotalAmount = grupedVoucherLines => (
+  grupedVoucherLines.map(({ amount }) => amount).reduce((acc, amount) => acc + amount)
```

Comment:
shouldn't you provide an initial value for `reduce`?

Reply:
An initial value is optional in this function

Reply:
yes, but in case of empty array you'll have result as `undefined`